migrations/20180330105350_rename-posts-to-authors.js

```diff
@@ -0,0 +1,15 @@
+exports.up = async knex => {
+  await knex.schema.renameTable('authors_posts', 'authors_articles');
+  await knex.schema.renameTable('issues_posts_order', 'issues_articles_order');
+  await knex.schema.alterTable('issues_articles_order', table => {
```

Comment:
If the column is named `article_order`, I think the table should be called `issues_article_order` or even `issue_article_order`.